integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java

```diff
@@ -216,7 +217,9 @@ void testRestartOperatorAndVerifyScaling() {
    * restartVersion are updated, and the server pods are recreated, the server pods' weblogic.domainRestartVersion
    * label is updated, and the new credentials are valid and can be used to access WebLogic RESTful
    * Management Services.
+   * Disabled due to bug.
    */
+  @Disabled
```

Comment:
Why disabled in cluster-reference branch only. Can  tag the JIRA accordingly.  We should not disable the test due to existing BUG

Reply:
re-enabled the test